syntax/element_content.ml

```diff
@@ -228,6 +228,8 @@ let script ~lang ~loc ~name children =
     [Nolabel, child]
   | _ -> Common.error loc "%s can have at most one child" name
 
+let textarea = script
+
```

Comment:
A link to the spec in comment would be nice too.

Reply:
Done; too bad I can't link the _Content model_ part.